### PR TITLE
add Next app for marmalade landing website to monorepo and update Next version

### PR DESCRIPTION
**Pull request by lkononenko**
## Reason:
Create Next app for new Marmalade landing pages website, add it to the monorepo and update Next and eslint-config-next versions
In progress https://app.asana.com/0/1204649083736950/1204769450652118/f 

## Changes made
- Created basic Next app for marmalade landing pages website with Typescript, eslint, app router and src folder setup
- Added this app (marmalade-website) to the monorepo
- Fixed packages versions
- Updated Next to 13.4.5 and eslint-config-next to match with new Next version (these updates are applicable only for kadena-docs, graph-client and marmalade-website) 
- Added test command to scripts to remark-plugin to fix tests failing

## Check off the following:

- [x] I have reviewed my changes and run the appropriate tests.
- [x] I have have run `rush change` to add the appropriate change logs.
- [x] I have added/edited docs.
- [x] I have added tutorials.
- [x] I have double checked and DEFINITELY added docs.

### CODE REVIEW

**Review by EnoF — APPROVED** [review]

File: packages/tools/remark-plugins/package.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
   "type": "module",
   "scripts": {
     "lint": "eslint src",
-    "build": "tsc"
+    "build": "tsc",
```
Comment (EnoF) [inline packages/tools/remark-plugins/package.json:9]: not sure if it will work out but you could try to use `heft build` (see other packages in the lib)

Comment (lkononenko) [reply packages/tools/remark-plugins/package.json:9]: @webpro could you please check comment from Andy regarding the build as you worked on this plugin. I just added the `test` command to fix failing of the tests

File: packages/apps/marmalade-website/.gitignore [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```
[the remark this answers is not in the document]
Comment (lkononenko) [reply packages/apps/marmalade-website/.gitignore]: good one! just removed the project related .gitignore and updated to use the global

File: packages/apps/marmalade-website/.eslintrc.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "next/core-web-vitals"
+}
```
Comment (eileenmguo) [inline packages/apps/marmalade-website/.eslintrc.json:3]: This should extend our @kadena-dev/eslint-config next config. I recommend referencing the kadena docs .eslintrc.json!

File: packages/apps/marmalade-website/package.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@kadena/marmalade-website",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "test": "echo \"No tests yet.\""
```
Comment (eileenmguo) [inline packages/apps/marmalade-website/package.json:10]: We have some kadena specific package json configurations. We used to be able to look at `lib/bootstrap-lib`, but I'm not sure if that's up to date. @alber70g Would you still recommend referencing that app for project setup?

File: packages/apps/marmalade-website/tsconfig.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+        "name": "next"
+      }
+    ],
+    "paths": {
+      "@/*": ["./src/*"]
+    }
+  },
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
+  "exclude": ["node_modules"]
+}
```
Comment (eileenmguo) [inline packages/apps/marmalade-website/tsconfig.json:28]: We also have a kadena specific tsconfig setup in @kadena-dev/heft-rig that you can extend and then remove all duplicate options

File: packages/apps/marmalade-website/.eslintrc.json (hunk 3 above)
Comment (lkononenko) [reply packages/apps/marmalade-website/.eslintrc.json:3]: yeah I planed to do it as a next step as in this pr some versions updates that are related to other packages was opening this initial pr with Next as it is

**Pull request closed without merging**